plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -204,3 +208,22 @@ func defaultPriorities() sets.String {
 		),
 	)
 }
+
+func GetEquivalencePod(pod *api.Pod) interface{} {
+	equivalencePod := EquivalencePod{}
+	podSpec := &(pod.Spec)
+	for _, vol := range podSpec.Volumes {
+		equivalencePod.Volumes = append(equivalencePod.Volumes, vol)
+	}
+	equivalencePod.NodeSelector = podSpec.NodeSelector
+	equivalencePod.Request = predicates.GetResourceRequest(pod)
+	equivalencePod.Ports = predicates.GetUsedPorts(pod)
+	return &equivalencePod
+}
+
+type EquivalencePod struct {
```

Comment:
A quick note:

namespace
labels
some annotations (PodAffinity, and others?)

How to enforce that when someone will add a new predicate/priority this will be mirrored here...